kuka_kr10_support/urdf/kr10r1420_macro.xacro

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+  <xacro:include filename="$(find kuka_resources)/urdf/common_constants.xacro"/>
```

Comment:
```suggestion
```
It seems this is no longer needed (was used for rad-degree conversion)

Reply:
removed it, also in the KR10 R1100 SIXX.